bot/exts/utils/snekbox.py

```diff
@@ -344,8 +344,15 @@ async def send_job(
                 log.trace("Formatting output...")
                 output, paste_link = await self.format_output(results["stdout"])
 
+            warning_message = ""
+
+            # This is done to make sure the last line of output contains the error
+            # and the error is not manually printed by the author with a syntax error.
+            if results["stdout"].rstrip().endswith("EOFError: EOF when reading a line") and results["returncode"] == 1:
+                warning_message += ":warning: Note: `input` is not supported by the bot :warning:\n\n"
+
```

Comment:
should this not be a part of get_results_message?

Reply:
Then we would aslo need to probably return the `warning_message` as well, which doesn't suits the function to return a string specifically for one outcome when it can be better sorted elsewhere.